vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogTestPage.java

```diff
@@ -169,4 +170,21 @@ private void createDivInDialog() {
         div.setSizeFull();
         add(button);
     }
+
```

Comment:
Shouldn't the `addResizeListener` api be tested?

Reply:
Added a test for the listener.

Reply:
The test should probably test the event's payload, not only that it was invoked.

Reply:
Added the values received from the event to the test check.